generatedTypes/incubator/WheelPicker/index.d.ts

```diff
@@ -28,6 +29,18 @@ export interface WheelPickerProps {
      * Row text style
      */
     textStyle?: TextStyle;
+    /**
```

Comment:
I'm not sure we supposed to do any changes in this files (it's under `generatedType`)

Reply:
I didn't, It's automatic :)